docs/getting-started/first-service.md

```diff
@@ -2,10 +2,11 @@
 !!! tip
     Hit ++"."++ on your keyboard to move forward in the tutorial. Use ++","++ to go back at any time.
 
-==**In this tutorial, you will deploy a "Hello world" service.**==
+**In this tutorial, you will deploy a "Hello world" service.**
+
 This service will accept an environment variable, `TARGET`, and print "`Hello ${TARGET}!`."
 
-For those of you familiar with other **source-to-url** tools, this may seem familiar. However, since our "Hello world" Service is being deployed as a Knative Service, it gets some **super powers (scale-to-zero, traffic-splitting) out of the box** :rocket:.
+Since our "Hello world" Service is being deployed as a Knative Service (not a Kubernetes Service), it gets some **super powers out of the box** :rocket:.
```

Comment:
Because our "Hello world" Service is being deployed as a Knative Service, not a Kubernetes Service, it gets some **super powers out of the box** :rocket:.